package.json

```diff
@@ -48,16 +45,13 @@
     "graphql": "^16.8.1",
     "graphql-request": "5.1.0",
     "graphql-tag": "^2.12.6",
-    "html-react-parser": "^3.0.8",
```

Comment:
This is used [above](https://github.com/pokt-foundation/portal-platform/pull/466/files#diff-a304fe5d954a4133a6883520bb452a78936c511eb02c5af810b99a59c62b408eR3) but removed here

Reply:
i thought I removed that import. we shouldn't need it